src/main.js

```diff
@@ -67,13 +68,13 @@ function viewSavedCovers() {
 }
 function showHomeCover() {
   makeNewFormLocation.classList.add('hidden');
-  viewSavedCoverLocation.classList.add('hidden');
+  viewSavedViewLocation.classList.add('hidden');
   randomCoverButton.classList.remove('hidden');
   saveCoverButton.classList.remove('hidden');
   homeButton.classList.add('hidden');
   homePageLocation.classList.remove('hidden');
 }
-function storeUserData(event) {
+function storeUserData() {
   currentCover = new Cover (userCover.value, userTitle.value, userDesc1.value, userDesc2.value);
   event.preventDefault();
```

Comment:
If you are preventing default behavior, best to move this to the top line of the function for clarity and to ensure the default behavior is actually prevented